mcc/Dispatcher.c

```diff
@@ -190,6 +196,7 @@ static IPTR mNew(struct IClass *cl, Object *obj, struct opSet *msg)
             // they were set during OM_NEW
             mSet(cl, obj, (struct opSet *)msg);
             data->visual_y = 1;
+            data->xpos     = 0;
```

Comment:
remove added spaces before `=` - no need to indent to the same level like the line before.